lib/udap_security_test_kit/client_suite/access_group.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'access_ac_interaction_test'
+require_relative 'access_cc_interaction_test'
+require_relative 'authorization_request_verification_test'
+require_relative 'token_request_ac_verification_test'
+require_relative 'token_request_cc_verification_test'
+require_relative 'token_use_verification_test'
+
+module UDAPSecurityTestKit
+  class UDAPClientAccess < Inferno::TestGroup
+    id :udap_client_access
+    title 'Client Access'
+    description %(
+      During these tests, the client system will access Inferno's simulated
+      FHIR server by requesting an access token using UDAP and making a FHIR
+      request presenting the access token. Inferno will then verify that any requests
+      made as a part of obtaining the token were conformant and that a token returned
+      from a token request was used on an access request.
+    )
+
+    run_as_group
+
+    test from: :udap_client_access_ac_interaction,
+         required_suite_options: {
+           client_type: UDAPClientOptions::UDAP_AUTHORIZATION_CODE
```

Comment:
Similar to my comment on the SMART App Launch PR, is there a reason not to have separate authorization code and client credentials groups and handle the options when the groups are imported?

Reply:
updated in [92fd83e](https://github.com/inferno-framework/udap-security-test-kit/pull/32/commits/92fd83e7e84e95668201a0abcd81af7599424286)

Reply:
factored out the access group in [10d0788](https://github.com/inferno-framework/udap-security-test-kit/pull/32/commits/10d0788c3a7bcd24f5cb33412b886299057b1098)